paparazzi/src/main/java/app/cash/paparazzi/SnapshotVerifier.kt

```diff
@@ -46,7 +46,16 @@ public class SnapshotVerifier @JvmOverloads constructor(
           throw AssertionError("File $expected does not exist")
         }
 
-        val goldenImage = ImageIO.read(expected)
+        val goldenImage = ImageIO.read(expected) ?: throw NullPointerException(
+          """
+          Failed to read the snapshot file from the file system.
+
+          If your project uses git LFS, it's possible that it's misconfigured on your machine and
+          Paparazzi has just loaded a pointer file instead of the real snapshot file. Follow git
+          LFS troubleshooting instructions and try again.
+
```

Comment:
```suggestion
          Failed to read the snapshot file.
```

yea, the NPE wasn't helpful as is, but, imo, using git lfs is an implementation detail, so probably best to address the better readability issue as simple as possible?

Reply:
TBH I don't think this is significantly more helpful than no message at all, i.e., when I looked at the line of code that threw the NPE I immediately understood that the read failed. What I spent half an hour on is realizing that the read failed because my git LFS was misconfigured. 

Definitely agree that git LFS is an implementation detail, and some teams will not be using it, but it seems that many likely will, and this message might save a lot of people a lot of time?

Alternatively we can of course publish an internal troubleshooting guide and point engineers to it, but having the likely cause of an issue already mentioned in the error message feels like a good way to reduce the time to troubleshoot.

Reply:
Fair point on the saving others from potential exploration time.  Ok, let's go with it.  We can always update if other users not using git lfs are confused by the verbiage.